src/pages/dapps.tsx

```diff
@@ -126,252 +116,154 @@ import wallet from "@/public/images/wallet.png" // width=300
+  <Flex className="justify-center" {...props} />
 )
 
 const Subtitle = (props: ChildOnlyProp) => (
-  <Text
-    fontSize={{ base: "xl", lg: "2xl" }}
-    lineHeight="140%"
-    color="text200"
-    mt={4}
+  <p
+    className="mb-6 mt-4 text-xl leading-xs text-body-medium lg:text-2xl"
     {...props}
   />
 )
 
 const Row = (props: ChildOnlyProp) => (
-  <Flex
-    w="full"
-    direction={{ base: "column", lg: "row" }}
-    align="flex-start"
-    {...props}
-  />
+  <Flex className="w-full flex-col items-start lg:flex-row" {...props} />
 )
 
 const IntroRow = (props: ChildOnlyProp) => (
   <Flex
-    w="full"
-    direction={{ base: "column", lg: "row" }}
-    align="flex-start"
-    bg="background.base"
-    p={8}
-    borderRadius="32px"
+    className="w-full flex-col items-start rounded-3xl bg-background p-8 lg:flex-row"
     {...props}
   />
 )
 
 const TwoColumnContent = (props: ChildOnlyProp) => (
   <Flex
-    w="full"
-    direction={{ base: "column", lg: "row" }}
-    align="flex-start"
-    me={{ lg: 8 }}
+    className="w-full flex-col items-start lg:me-8 lg:flex-row"
     {...props}
   />
 )
 
 const StyledH2 = (props: ChildOnlyProp) => (
-  <OldHeading
-    fontSize="2xl"
-    lineHeight="22px"
-    letterSpacing={0}
-    mt={2}
-    {...props}
-  />
+  <h2 className="mb-8 mt-2 text-2xl" {...props} />
 )
 
-const H2 = (props: HeadingProps) => (
-  <Heading
-    mt={12}
-    mb={8}
-    fontSize={{ base: "2xl", md: "2rem" }}
-    fontWeight="semibold"
-    lineHeight={1.4}
+const H2 = (props: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <h2
+    className={cn(
+      "mb-8 mt-12 text-2xl font-semibold md:text-3xl",
+      props.className
+    )}
     {...props}
   />
 )
 
-const H3 = (props: HeadingProps) => (
-  <OldHeading
-    as="h3"
-    fontSize={{ base: "xl", md: "2xl" }}
-    fontWeight="semibold"
-    lineHeight={1.4}
-    {...props}
-  />
+const H3 = (props: BaseHTMLAttributes<HTMLHeadingElement>) => (
+  <h3 className="mb-4 mt-8 text-xl font-semibold md:text-2xl" {...props} />
 )
 
 const StyledH3 = (props: ChildOnlyProp) => (
-  <Heading
-    as="h3"
-    lineHeight={1.4}
-    fontSize="xl"
-    fontWeight="bold"
-    mb={2}
-    mt={6}
-    sx={{
-      a: {
-        dispalay: "none",
-      },
-    }}
-    {...props}
-  />
+  <h3 className="mb-2 mt-6 text-xl font-bold" {...props} />
 )
 
 const StyledInfoBanner = (props: ComponentPropsWithRef<typeof InfoBanner>) => (
   <InfoBanner w={{ lg: "50%" }} {...props} />
 )
 
 const Column = (props: ChildOnlyProp) => (
-  <Box flex="1 1 75%" mb={6} me={{ lg: 8 }} {...props} />
+  <div className="mb-6 flex-[1_1_75%] lg:me-8" {...props} />
 )
 
 const FullWidthContainer = (
   props: ChildOnlyProp & { ref: React.RefObject<HTMLDivElement> }
 ) => (
   <Page
-    m={0}
-    mb={16}
-    pt={16}
-    pb={8}
-    borderTop="1px solid"
-    borderColor="border"
-    bg="ednBackground"
+    className="m-0 mb-16 border-t border-border bg-background-highlight pb-8 pt-16"
```

Comment:
removing the border color as it is redundant here. The default color is going to be that one.
```suggestion
    className="m-0 mb-16 border-t bg-background-highlight pb-8 pt-16"
```